style.css

```diff
@@ -0,0 +1,294 @@
+/* stylelint-disable CssSyntaxError */
+/* stylelint-disable no-descending-specificity */
```

Comment:
- [ ] Linter is our red pen when writing codes, you should not disable a linter because it is to help you well align and dry your source code, please remove this commented lines  and work to fix the issues